UNRELEASED-V4.md

```diff
@@ -48,39 +53,50 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Code quality
 
-- Enabled react/no-unsafe with checkAliases ([#1695](https://github.com/Shopify/polaris-react/pull/1695))
-- Added hooks to storybooks scope ([#1672](https://github.com/Shopify/polaris-react/pull/1672))
-- Removed all uses of `ReactDOM.findDOMNode` ([#1696](https://github.com/Shopify/polaris-react/pull/1696))
-- Simplified `WithinContentContainer` context type ([#1602](https://github.com/Shopify/polaris-react/pull/1602))
-- Removed test errors and non-deprecation warnings ([#1715](https://github.com/Shopify/polaris-react/pull/1715))
+- Ignored app bridge deprecation warnings in tests ([#1852](https://github.com/Shopify/polaris-react/pull/1852))
+- Updated withAppProvider` to use a functional component rather than a class component ([#1813](https://github.com/Shopify/polaris-react/pull/1813))
```

Comment:
```suggestion
- Updated `withAppProvider` to use a functional component rather than a class component ([#1813](https://github.com/Shopify/polaris-react/pull/1813))
```